src/interface/lazy.jl

```diff
@@ -7,12 +7,13 @@ const AbstractArrayOrBroadcasted = Union{AbstractArray,Broadcasted}
 mutable struct LazyTensor{T, N}
     data
     extrude::NTuple{N, Bool}
+    size::NTuple{N, UInt64}
```

Comment:
This field should be renamed `shape` for consistency with other naming choices in the codebase